tests/snappi_tests/multidut/pfc/test_m2o_oversubscribe_lossy.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+import random
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts, \
+    fanout_graph_facts                                                                          # noqa: F401
+from tests.common.snappi_tests.snappi_fixtures import snappi_api_serv_ip, snappi_api_serv_port, \
+    snappi_api, snappi_dut_base_config, get_tgen_peer_ports, get_multidut_snappi_ports, \
+    get_multidut_tgen_peer_port_set, cleanup_config                                             # noqa: F401
+from tests.common.snappi_tests.qos_fixtures import prio_dscp_map, \
+    lossless_prio_list                                                                          # noqa: F401
+from tests.snappi_tests.variables import config_set, line_card_choice
+from tests.snappi_tests.multidut.pfc.files.m2o_oversubscribe_lossy_helper import run_pfc_m2o_oversubscribe_lossy_test
+from tests.common.snappi_tests.snappi_test_params import SnappiTestParams
+
+pytestmark = [pytest.mark.topology('multidut-tgen')]
+
+
+@pytest.mark.parametrize('line_card_choice', [line_card_choice])
+@pytest.mark.parametrize('linecard_configuration_set', [config_set])
+def test_m2o_oversubscribe_lossy(snappi_api,                                  # noqa: F811
+                                 conn_graph_facts,                            # noqa: F811
+                                 fanout_graph_facts,                          # noqa: F811
+                                 line_card_choice,
+                                 duthosts,
+                                 prio_dscp_map,                               # noqa: F811
+                                 lossless_prio_list,                          # noqa: F811
+                                 linecard_configuration_set,
+                                 get_multidut_snappi_ports,):                 # noqa: F811o
+    """
+    Run PFC watchdog test under many to one traffic pattern
```

Comment:
please fix comments as this is not pfcwd test instead oversubscription test

Reply:
this has been fixed